h/templates/app.html.jinja2

```diff
@@ -29,7 +29,7 @@
            on-clear="search.clear()"></div>
       <!-- / Searchbar -->
 
-      <group-list class="group-list" ng-if="auth.user && feature('groups')">
+      <group-list class="group-list" ng-if="auth.user && feature('groups') && isEmbedded">
```

Comment:
I'm not sure that's what you want. You yourself added `isSidebar` not so long ago: 255a820.